lerna-debug.log

```diff
@@ -0,0 +1,13 @@
+0 silly argv {
+0 silly argv   _: [ 'add' ],
+0 silly argv   scope: '@booking-services/apigateway',
+0 silly argv   globs: [],
+0 silly argv   lernaVersion: '3.22.1',
+0 silly argv   '$0': 'lerna',
+0 silly argv   pkg: '@booking-services/useraccount'
+0 silly argv }
+1 notice cli v3.22.1
+2 verbose rootPath /home/pedro/Platzi/booking-services
+3 info versioning independent
+4 error Error: 404 Not Found - GET https://registry.npmjs.org/@booking-services%2fuseraccount - Not found
+4 error     at /home/pedro/Platzi/booking-services/node_modules/@evocateur/npm-registry-fetch/check-response.js:104:15
```

Comment:
Is relevant to maintain this file in the repo?

Reply:
not really, I left it in a comit before creating the .gitignore. i will delete it.